### PR TITLE
[BLOG] Create Announcing Scientific Python Accessibility Events

### PR DESCRIPTION
**Pull request by isabela-pf**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] Blog post featured image is in PNG or JPEG format, **not** SVG.
- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/announcing-scientific-python-accessibility-events.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## What’s happening at these events?
+
+Each event has its own theme and will cover different, though overlapping, accessibility considerations. You can attend one or both events, no prerequisites required.
+
+### Accessibility in your digital day-to-day
+
+For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks. Advice, if you can find it all, is often vague enough or detailed enough to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!
```
Comment (trallard) [inline apps/labs/posts/announcing-scientific-python-accessibility-events.md:23]:
```suggestion
For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks, including how to get started. If you can find it all, advice is often vague enough or detailed enough to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!
```

**Comment by isabela-pf** [comment]
Rewording accepted and registration links added. Please let me know if you notice any other issues.

File: apps/labs/posts/announcing-scientific-python-accessibility-events.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## What’s happening at these events?
+
+Each event has its own theme and will cover different, though overlapping, accessibility considerations. You can attend one or both events, no prerequisites required.
+
+### Accessibility in your digital day-to-day
+
+For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks, including how to get started. If you can find it all, advice is often vague enough or detailed enough to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!
```
Comment (gabalafou) [inline apps/labs/posts/announcing-scientific-python-accessibility-events.md:23]:
```suggestion
For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks, including how to get started. If you can find it at all, advice is often vague enough (or so overwhelmingly detailed) to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!
```

File: apps/labs/posts/announcing-scientific-python-accessibility-events.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+**When:** Tuesday, 17 September, 2024 at 8 AM Pacific Time ([in your timezone](https://arewemeetingyet.com/Los%20Angeles/2024-09-17/08:00/Accessibility%20in%20your%20digital%20day-to-day%20with%20Scientific%20Python%20and%20Quansight%20Labs))
+
+**What attendees should know:** No prior knowledge needed.
+
+**[Register for the first event here](https://us06web.zoom.us/meeting/register/tZIvd-qprjIiH90QqVSf_GKSQzqt_x1H21Ms)**
+
+### Data visualization and accessibility
+
+Data visualization is a varied and often complex topic that draws on many disciplines. Many have advised on ways to make data visualizations include more options for people who are blind and low vision and deaf and hard of hearing. This event will compile existing recommendations to provide an abbreviated guide to the what and why behind making data visualizations available to more people.
```
Comment (gabalafou) [inline apps/labs/posts/announcing-scientific-python-accessibility-events.md:33]:
```suggestion
Data visualization is a varied and complex topic that draws on many disciplines. Many have advised on ways to make data visualizations more inclusive of people who are blind or low vision and/or deaf or hard of hearing. This event will compile existing recommendations to provide an abbreviated guide to the what and why behind making data visualizations available to more people.
```

File: apps/labs/posts/announcing-scientific-python-accessibility-events.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: 'Announcing Scientific Python Accessibility Events'
+published: September 5, 2024
+authors: [isabela-presedo-floyd]
+description: 'I am happy to announce two upcoming public events focused on helping the scientific Python ecosystem develop their accessibility skills before the new year.'
+category: [Access-centered, OSS Experience]
+featuredImage:
+  src: /posts/announcing-scientific-python-accessibility-events/blog_feature_1.png
+  alt: 'A tree branch extends from the screen of a laptop with a bar chart.'
```
Comment (gabalafou) [inline apps/labs/posts/announcing-scientific-python-accessibility-events.md:9]:
```suggestion
  alt: 'A tree branch extends from a laptop screen. There is a bar chart on the screen.'
```

**Review by gabalafou — CHANGES_REQUESTED** [review]
I left a few copy suggestions, feel free to incorporate or not, as you think best.

There are a few unrelated changes in this PR that need to be removed before merging. I will fix it up for you.

**Comment by isabela-pf** [comment]
Just checking on this! I believe I've reviewed all the changes and that @gabalafou has kindly corrected the mis-caught commits. Please let me know if there's anything else I need to do to help the blog post progress.

**Review by trallard — APPROVED** [review]

**Review by gabalafou — APPROVED** [review]

**Pull request merged**